themes/learn.jquery.com/style.css

```diff
@@ -53,11 +53,11 @@ a {
 	top: 11px;
 	width: 120px;
 	background-color: #ffea94;
-	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E)); 
-	background-image: -webkit-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:    -moz-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:     -ms-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:      -o-linear-gradient(top, #ffea94, #FFEA5E); 
+	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E));
```

Comment:
Commits are best focused on what they describe. These changes have nothing to do with "removing templating for contributors attributions" so they shouldn't be included.

BTW, some Git-aware editors allow to automatically strip trailing spaces only for changed lines.